### DISCUSSION TITLE
versioning issues

### DISCUSSION THREAD

**Discussion by ahmed26399** [atrocore/atrocore#591] [Q&A] [answered]
Hello,
I am trying to use AtroCore. I want to know about versioning. i tried to make a version and i used type like  published. But every change i did a change the version is changed. How can i make a version that didn't changes
Thanks in advance

**Comment by rratsun** [comment]
The only thing that you should do is to create new release (new version) that bigger than current.
* Do some changes in code
* Create new release (https://nimb.ws/HuzPcj)
* Update your system

**Reply by ahmed26399** [reply]
Hello,
Thank you for your answer but I did not explain my request well. in fact, the versioning I'm talking about is having a snapshot of the current state at time x (time) that we can't modify at time x+1
Thanks in advance

**Comment by ahmed26399** [comment, marked as answer]
Hello,
Thank you for your answer but I did not explain my request well. in fact, the versioning I'm talking about is having a snapshot of the current state at time x (time) that we can't modify at time x+1
Thanks in advance

**Reply by atrolex** [reply]
This is currently not possible. We develop now a new paid module "Versioning". With this module this will be possible. The module with be released in February. As a workaround you can create copies of your products in some other catalog to fix the description state.
